services/API-service/src/scripts/seed-rainfall-data.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from '@nestjs/common';
+import { InterfaceScript } from './scripts.module';
+import { Connection } from 'typeorm';
+import { SeedHelper } from './seed-helper';
+import { RainfallTriggersEntity } from '../api/rainfall-triggers/rainfall-triggers.entity';
+
+@Injectable()
+export class SeedRainfallData implements InterfaceScript {
+  private connection: Connection;
+  private readonly seedHelper: SeedHelper;
+  private rainfallTriggersRepository;
+
+  public constructor(connection: Connection) {
+    this.connection = connection;
+    this.seedHelper = new SeedHelper(connection);
+  }
+
+  public async run(): Promise<void> {
+    this.rainfallTriggersRepository = this.connection.getRepository(
+      RainfallTriggersEntity,
+    );
+
+    // EGY
+    // Trigger levels per lat/lon pixel
+    const fileName = `./src/scripts/git-lfs/rainfall/rainfall_trigger_levels_EGY.csv`;
+    const data = await this.seedHelper.getCsvData(fileName);
+
+    await Promise.all(
+      data.map(
+        async (pixel): Promise<void> => {
+          return this.rainfallTriggersRepository
+            .createQueryBuilder()
+            .insert()
+            .values({
+              countryCode: 'EGY',
```

Comment:
@jannisvisser can we make this country agnostic? We could accept a `countryCodeISO3` parameter then we can support rainfall triggers for any country.

Reply:
done (when committed). and I'm only seeding this data for the right countries, based on a new country-property which is at the moment 'glofas' or 'rainfall' (with its own enum ;) )